style.css

```diff
@@ -0,0 +1,114 @@
+* {
+  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
```

Comment:
Your CSS rules are clear and easy to follow.  Push here is to use classes and ids as selectors to make it easier to see and where each of them is being used.  This code would then be easier to maintain for other devs to collaborate on.